src/Workspaces/Core/Portable/Differencing/Match.LongestCommonSubsequence.cs

```diff
@@ -17,12 +17,16 @@ internal LongestCommonSubsequence(Match<TNode> match)
                 _match = match;
             }
 
-            protected override bool ItemsEqual(IReadOnlyList<TNode> oldSequence, int oldIndex, IReadOnlyList<TNode> newSequence, int newIndex)
+            protected override bool ItemsEqual(
+                IReadOnlyList<TNode> oldSequence, int oldIndex, 
+                IReadOnlyList<TNode> newSequence, int newIndex)
             {
                 return _match.Contains(oldSequence[oldIndex], newSequence[newIndex]);
             }
 
-            internal Dictionary<TNode, TNode> GetMatchingNodes(IReadOnlyList<TNode> oldNodes, IReadOnlyList<TNode> newNodes)
+            internal Dictionary<TNode, TNode> GetMatchingNodes(
```

Comment:
Github's bad design should not be a measure for how we format code. You can get a Chrome extension to make github view wider.